### DISCUSSION TITLE
part-ii/2021-51/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#327] [Question comments]
Part II 2021 Dynamical Systems: Paper 1, Section II, A 

https://questions.tripos.org/part-ii/2021-51/

**Comment by eha1234** [comment]
(a) bookwork 

(b) Use $V = x^2/(1+x^2)^2+y^2/2$ and consider $y=0$ carefully. 

(d) Use the $V$ you found above for the first bit, for the second I argued by contradiction.
